feature/bgp/policybase/otg_tests/extcomm_action_test/README.md

```diff
@@ -0,0 +1,262 @@
+      * community-member = [ "500000:500000" ]
+      * match-set-options = ANY
+    * Create `ext-community-sets/ext-community-set[name='add_ext_comms']` with members as follows:
+      * community-member = [ "100:100000", "200000:200000" ]
+
+  * Create a `/routing-policy/policy-definitions/policy-definition/policy-definition/`
+    named 'add_ext_comms' with the following `statements/`
+    * statement[name='add_ext_comms']/
+      * actions/bgp-actions/set-ext-community/reference/config/ext-community-set-refs =
+          /routing-policy/defined-sets/bgp-defined-sets/ext-community-sets/ext-community-set[name='add_ext_comms']
+      * actions/bgp-actions/set-ext-community/config/options = ADD
+      * actions/config/policy-result = NEXT_STATEMENT
+    * statement[name='accept_all_routes']/
+      * actions/config/policy-result = ACCEPT_ROUTE
+
+  * Create a `/routing-policy/policy-definitions/policy-definition/policy-definition/`
+    named 'match_and_add_ext_comms' with the following `statements/`
+    * statement[name='match_and_add_ext_comms']/
+      * conditions/bgp-conditions/match-ext-community-set/config/ext-community-set =
+          /routing-policy/defined-sets/bgp-defined-sets/ext-community-sets/ext-community-set[name='match_ext_comms']
+      * actions/bgp-actions/set-ext-community/reference/config/ext-community-set-refs =
+          /routing-policy/defined-sets/bgp-defined-sets/ext-community-sets/ext-community-set[name='add_ext_comms']
+      * actions/bgp-actions/set-ext-community/config/options = ADD
+      * actions/config/policy-result = NEXT_STATEMENT
+    * statement[name='accept_all_routes']/
+      * actions/config/policy-result = ACCEPT_ROUTE
+
+  * For each policy-definition created, run a subtest (RT-7.8.3.x-<policy_name_here>) to
+    * Use gnmi Set REPLACE option for:
+      * `/routing-policy/policy-definitions` to configure the policy
+      * Use `/network-instances/network-instance/protocols/protocol/bgp/neighbors/neighbor/apply-policy/config/import-policy`
+        to apply the policy on the DUT bgp neighbor to the ATE port 1.
+    * Verify routes are received on ATE port 1 for accepted prefixes.
+    * Verify expected communities are present.
+
+    * Expected matches for each policy
+      |              | add_ext_comms                                               | match_and_add_ext_comms                                     |
+      | ------------ | ----------------------------------------------------------- | ----------------------------------------------------------- |
+      | prefix-set-1 | [ 100:100000, 200000:200000 ]                               | none                                                        |
+      | prefix-set-2 | [ 100:100000, 200000:200000, 500:500, 600:600 ]             | [ 500:500, 600:600 ]                                        |
+      | prefix-set-3 | [ 100:100000, 200000:200000, 500000:500000, 600000:600000 ] | [ 100:100000, 200000:200000, 500000:500000, 600000:600000 ] |
+
```

Comment:
Is your intention to test **ext-comunity** here? Seems to be OOS of this test.
if OOS - remove
if in scope, then you need something like `route-target:`, link-bakdwidt:` etc.

the 500:500 or 500000:500000 are NOT extended commuities. They may be Large communities, but this are not extended one.
https://www.iana.org/assignments/bgp-parameters/bgp-parameters.xhtml

Reply:
To maintain separation between std community and ext community, I used separate sub tests.  RT-7.8.2, RT-7.8.3 (covers only 8 byte communities scenario).

I propose that route-target, color, link-bandwidth communities would be in separate readmes. 

Agreed that I should move the link-bandwidth to a completely separate readme.